examples/app.py

```diff
@@ -85,7 +88,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
     CELERY_RESULT_BACKEND="cache",
     DEBUG=True,
-    SEARCH_UI_SEARCH_API='invenio_search_ui.api',
+    SEARCH_UI_SEARCH_API='invenio_records_rest.recid_list',
```

Comment:
I'm not sure what do you mean (please correct me if I'm wrong), but in the example app works fine. https://github.com/inveniosoftware/invenio-search-ui/blob/master/invenio_search_ui/templates/invenio_search_ui/search.html#L44